api/types/userloginstate/user_login_state.go

```diff
@@ -25,6 +25,14 @@ import (
 	"github.com/gravitational/teleport/api/types/trait"
 )
 
+const (
+	// OriginalRolesAndTraitsSet is an annotation that will indicate that the original roles and traits have been
+	// set for this objects. For existing user login states, this will not be set, indicating that we can't
+	// use these fields reliably.
+	// TODO(mdwn): Remove this in v17.
```

Comment:
We usually use a bit different form for this: `// DELETE IN 17 (mdwn)` , it's easier to find them all just by search if they are standardized.

Reply:
Will do!